### PR TITLE
Create a branch and add the sgx actions

### PR DESCRIPTION
**Pull request by jcape**
Copy over the actions from the sgx repo.

Phase 2 will be removing them from that repo and pointing it here...

### CODE REVIEW

**Comment by jcape** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/actions/1" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/actions/1?utm_source=stack-comment).

**Review by nick-mobilecoin — APPROVED** [review]

File: sgxsdk/action.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Install SGX SDK
+description: Installs the Intel SGX SDK (and propagates environment variables)
+
+runs:
+  using: composite
+  steps:
+    - name: Install SGX SDK
+      shell: bash
+      # In order to propagate teh environment variables to subsequent steps we
```
Comment (NotGyro) [inline sgxsdk/action.yaml]: Nitpick: teh -> the

**Review by NotGyro — APPROVED** [review]
Looks good to me.

File: sgxsdk/action.yaml (hunk 1 above)
Comment (jcape) [reply sgxsdk/action.yaml]: Fixed, ty

**Pull request merged**